trackbasket_be/resources/at_risk_user.py

```diff
@@ -44,4 +45,23 @@ def patch(self, id):
       return {'data': { 'id': 'at_risk_user', 'attributes': {'error': "User not found"} } }, 400
     db.session.commit()
     return at_risk_user.json(), 200
+  
+  def delete(self, id):
+    at_risk_user = db.session.query(AtRiskUser).filter_by(at_risk_user_id=id).first()
+    if at_risk_user:
+      for lst in at_risk_user.shopping_lists:
```

Comment:
iterate through at risk user's lists and delete

Reply:
I don't see what changed compared to what you showed me earlier